src/Containers/PictureCard.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import styled, { DefaultTheme } from 'styled-components';
+import { StyledIcon } from 'styled-icons/types';
+import { flex, position } from '@Utils/Mixins';
+import { Card as C } from './Card';
+
+interface TagProps {
+    icon: StyledIcon;
+    text: string;
+}
+
+
+interface PictureCardProps extends MainInterface, ResponsiveInterface {
+    image: string;
+    tags?: TagProps[];
+    theme?: DefaultTheme;
+    alt?:  string;
+    height?: string;
+    width?:  string;
+    borderradiustop?: string;
```

Comment:
Whats the use case where we would need to toggle radius of top and bottom?

Reply:
well yes the default border-radius which I had put on the code was 25px for all sides, but I got this component which I needed to use PictureCard in it but in a way which the top had a border-radius but the bottom needed to be flat so I had to put it as a prop